crates/btc-relay/src/lib.rs

```diff
@@ -54,8 +54,8 @@ use mocktopus::macros::mockable;
 #[cfg(feature = "runtime-benchmarks")]
 use bitcoin::types::{BlockBuilder, TransactionBuilder, TransactionOutput};
 use bitcoin::{
-    merkle::{MerkleProof, ProofResult},
-    types::{BlockChain, BlockHeader, H256Le, Transaction, Value},
+    merkle::{PartialTransactionProof, ProofResult},
+    types::{BlockChain, BlockHeader, FullTransactionProof, H256Le, Transaction, Value},
```

Comment:
Might be nice to re-export these new types to make downstream usage easier

Reply:
Downstream should probably use metadata-generated types but I re-exported anyway